hr_timesheet_adj/models/account_analytic_line.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    task_id = fields.Many2one(ondelete='restrict')
```

Comment:
@yostashiro Do you think we should also add task_id here as well?

Reply:
@TimLai125 Yes, I think that's probably safer.